packages/mock-account-provider/src/requesters.ts

```diff
@@ -30,13 +35,6 @@ function testGraphqlElementSuccess(el: GraphqlResponseElement) {
   })
 }
 
-function isGraphqlArray(a: GraphqlResponseElement | GraphqlResponseArray) {
-  if (_.isArray(a)) {
-    return a as GraphqlResponseArray
-  }
-  return a as GraphqlResponseElement
-}
-
 export async function graphqlQuery(options: GraphqlQueryConfig) {
```

Comment:
Do we still use this or can it be removed now?

Reply:
removed